.github/workflows/reviewdog.yml

```diff
@@ -92,8 +94,8 @@ jobs:
           cat temp_golangci-lint-cgo.txt | reviewdog \
           -f=golangci-lint \
           -name="Lint Warnings" \
-          -reporter=github-check \
-          -filter-mode=added \
+          -reporter=github-pr-check \
+          -filter-mode=nofilter \
```

Comment:
not sure about nofilter, could be [too much](https://github.com/reviewdog/reviewdog?tab=readme-ov-file#filter-mode) output?